json-2/src/main/java/com/baeldung/jsonld/serialization/hydrajsonld/HydraJsonldSerializationExample.java

```diff
@@ -0,0 +1,52 @@
+package com.baeldung.jsonld.serialization.hydrajsonld;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.BeanDescription;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationConfig;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.databind.ser.BeanSerializerModifier;
+import com.fasterxml.jackson.databind.ser.std.BeanSerializerBase;
+
+import de.escalon.hypermedia.hydra.serialize.JacksonHydraSerializer;
+
+public class HydraJsonldSerializationExample {
+    public static void main(String[] args) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        mapper.registerModule(getJacksonHydraSerializerModule());
+
+        HydraJsonldExamplePerson person = new HydraJsonldExamplePerson();
+        person.id = "http://example.com/person/1234";
+        person.name = "Example Name";
+
+        System.out.println(mapper.writerWithDefaultPrettyPrinter()
+            .writeValueAsString(person));
+    }
+
+    public static SimpleModule getJacksonHydraSerializerModule() {
+        return new SimpleModule() {
```

Comment:
There is a warning here about serial version Ids

Reply:
I must have some settings not properly set because I see no warnings. Could you point me in the right direction?

Reply:
The warning here is - The serializable class does not declare a static final serialVersionUID field of type long. But I think we can leave this one. Don't worry.